### PR TITLE
Formatting and hash-bangs

### PR DESCRIPTION
**Pull request by jsuntheimer72**
Just added #!/bin/bash to build files that lacked it, and listed the packages one-per-line.

### CODE REVIEW

File: Slax/debian/modules/02-xorg/build [hunk 1]
```diff
@@ -1,8 +1,19 @@
+#!/bin/bash
 
 apt-get update
-apt-get install --yes --no-install-recommends xserver-xorg xinit xterm \
-blackbox libxcursor1 breeze-cursor-theme x11-utils wmctrl xdotool \
-libdrm-intel1 libgl1-mesa-dri libglu1-mesa
+apt-get install --yes --no-install-recommends
```
Comment (Tomas-M) [inline Slax/debian/modules/02-xorg/build]: missing \

**Comment by jsuntheimer72** [comment]
I fixed the missing slash.  This is my first pull request (!), so I'm not sure if there's anything further I must do.  I pushed the change to master, and I see my commit message above this.

**Comment by Tomas-M** [comment]
Thanks

**Comment by Tomas-M** [comment]
forward slashes in-apps build ......... grr ..... :)

**Pull request merged**